### ISSUE TITLE
[Feature Request] Along w/ lovelaces, Value should allow to get assets only

### ISSUE THREAD

**Issue by nemo83** [Hyperion-BT/helios#61]
Currently `Value` has a `get_lovelace()` to get the lovelace value, but there is not `get_assets` or equivalent, where only native assets are returned.

I've currently worked around it w/
```javascript
Value::from_map(myValue.to_map().filter((a: MintingPolicyHash, _) -> { a != MintingPolicyHash::new(#) }));
```

**Comment by christianschmitz** [comment]
Thanks for calling this out! I've just released v0.12.12 which implements `value.get_assets()`. Please take a look

**Comment by nemo83** [comment]
> Thanks for calling this out! I've just released v0.12.12 which implements `value.get_assets()`. Please take a look

All working. Thanks.
